Extraction/Classes/ARLoadFailureView.m

```diff
@@ -80,6 +80,7 @@ - (instancetype)initWithFrame:(CGRect)frame;
 
 - (IBAction)retry:(id)sender;
 {
+  self.shouldRotate = YES;
   [self rotate];
```

Comment:
2) On the 2nd retry attempt this was not changed and `stopRotating` was still `YES`.